pom.xml

```diff
@@ -13,17 +13,12 @@
   <description>Adds the ability to wrap your job with pre- and post- build steps which are only executed when a manual release build is triggered.</description>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/Release+Plugin</url>
 
-  <build>
-    <plugins>
-      <plugin>
-        <artifactId>maven-release-plugin</artifactId>
-        <version>2.5.1</version>
-        <configuration>
-          <goals>deploy</goals>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
+  <properties>
+    <jenkins.version>1.609.3</jenkins.version>
```

Comment:
Ok.

@avalanche123 not sure if you are the current maintainer but you seem to have done the last releases. Are you acting as maintainer?